ui/lib/path.js

```diff
@@ -43,6 +43,7 @@ export const registerProject = id => `/projects/${id}/register`;
 export const wallet = _params => "/wallet";
 export const profile = _params => "/profile";
 export const help = _params => "/help";
+export const registerUser = _params => "/register-user";
```

Comment:
Is this path temporary? I'd expect building up a namespace similar to `projects`, i.e.: `/users/register`

Reply:
It's temporary until the general routing regarding users/orgs is figured out.

Reply:
Makes perfect sense!